specification/frontdoor/resource-manager/readme.nodejs.md

```diff
@@ -0,0 +1,15 @@
+## Node.js
+
+These settings apply only when `--nodejs` is specified on the command line.
+Please also specify `--node-sdks-folder=<path to root folder of your azure-sdk-for-node clone>`.
+
+``` yaml $(nodejs)
+nodejs:
+  azure-arm: true
+  package-name: azure-arm-frontdoor
+  package-version: 5.2.0
+  generate-license-txt: true
+  generate-package-json: true
+  generate-readme-md: false
```

Comment:
This looks like a new service. In that case, I recommend replacing these three properties (`generate-license-txt`, `generate-package-json`, and `generate-readme-md`) to just `generate-metadata: true`.